data/hails.txt

```diff
@@ -1002,12 +1002,197 @@ phrase "friendly civilian"
+		"decided to argue with a Navy officer"
+	word
+		""
+		""
+		" a few years ago"
+		" a few months ago"
+		" a few weeks ago"
+	word
+		" and they were never seen again"
+		" and never came back"
+		" and mysteriously disappeared"
+		", never to be seen again"
+	word
+		". "
+	word
+		"I wonder what happened to them."
+		"The universe is a dangerous place!"
+		"I'm not surprised."
+
+phrase "friendly civilian"
+	word
+		"I love it when "
+	word
+		"I'm walking through the spaceport"
+		"I land on a planet"
+		"I'm browsing the outfitter"
+		"I'm browsing the shipyard"
+	word
+		" and I "
+	word
+		"find interesting jobs"
+		"meet interesting people"
+		"get offered a unique job"
+		"find a unique quest"
+		"find things that I wouldn't find elsewhere"
+	word
+		"."
+
+phrase "friendly civilian"
+	word
+		"I have heard crazy stories about"
+		"My mother use to tell me a story about"
+		"My father use to tell me a story about"
+		"My friend told me a story about"
+		"Some people like to spin tales of"
+		"This old drunk I met talked about"
+	word
+		" "
+	word
+		"giant"
+		"scary"
+		"friendly"
+		"hostile"
+		"strange"
+		"sentient"
+		"intelligent"
+	word
+		" "
+	word
+		"rodent"
+		"chipmunk"
+		"rat"
+		"squirrel"
+		"mouse"
+		"beaver"
+		"gopher"
+		"rabbit"
+	word
+		" "
+	word
+		"creatures"
+		"aliens"
+		"beings"
+		"things"
+	word
+		" that "
+	word
+		"are more advanced than humans"
+		"have developed the hyperdrive"
+		"can terraform planets better than humans"
+		"could crush the Navy if they wanted to"
+		"live near humanity"
+		"are hiding in plain sight"
+	word
+		". "
+	word
+		""
+		"Could they be real?"
+		"How could anyone believe that?"
+		"Where could they be?"
+		"Well where are they?"
+		"What a crazy story, right?"
+		"I'll believe it when I see it."
+		"I wonder what they were smoking."
+		"I'm ashamed that I ever believed it."
+
+# Hint toward the source or plot of a few transport missions
+phrase "friendly civilian"
+	word
+		"Never transport someone from the Paradise Worlds. Their dog will probably throw up in your bunk rooms."
+		"I hear that most of the crew on pirate ships are just teenagers trying to make enough money to leave. That must be a terrible life."
+		"Apparently scientists near Wei are saying that the ice age caused by the supervolcano is ending soon."
+		"Have you seen any Wolf-Rayet stars yet? I like to just stay in the system and stare at them for hours."
+		"I've been told that there is some weird stuff that goes on in the Deep."
+		"There was this one time when I was lucky enough to find There Might Be Riots hanging out at a spaceport. They even asked me to transport them to one of their concerts!"
+		"A buddy of mine was busted for transporting a few tons of drugs for a shady character. They should have known better."
+		"It's amazing how far we have come as a species in the past one thousand years. Not to long ago in the time of the universe, we couldn't even terraform planets!"
```

Comment:
Can we get some of these phrasings turned into contractions? "It's amazing how far we have come" sounds awkward to me without a contraction, for example.